meta-balena-common/recipes-support/os-helpers/os-helpers/os-helpers-tpm2

```diff
@@ -23,6 +23,9 @@ EFI_GLOBAL_VARIABLE_GUID="8be4df61-93ca-11d2-aa0d-00e098032b8c"
 
 TPM_EVENTLOG_PATH="/sys/kernel/security/tpm0/binary_bios_measurements"
 
+# silence errors messages from trying unsupported TCTI backends
+export TPM2TOOLS_TCTI=device
```

Comment:
I think this should be `device:/dev/tpmrm0`, the default is `/dev/tpm0`. The `rm` stands for resource manager, which implements kind of a "virtual memory" for the TPM. Without one and with enouth operations, it is possible to clog the TPM rendering it unusable until a powercycle. That is what `abrmd` was used for before this was properly implemented in the kernel.